Samples/NotifyUsers/AppBackend/AuthenticationTestHandler.cs

```diff
@@ -45,13 +41,13 @@ public class AuthenticationTestHandler : DelegatingHandler
             return base.SendAsync(request, cancellationToken);
         }
 
-        private bool verifyUserAndPwd(string user, string password)
+        private static bool VerifyUserAndPwd(string user, string password)
         {
             // This is not a real authentication scheme.
             return user == password;
         }
 
-        private Task<HttpResponseMessage> Unauthorized()
+        private static Task<HttpResponseMessage> Unauthorized()
```

Comment:
Can you update this to return `Task.FromResult(response)` ? And any other place where we use `TaskCompletionSource` in this manner?